.config/nextest.toml

```diff
@@ -1,2 +1,18 @@
 [profile.default]
+# "retries" defines the number of times a test should be retried. If set to a
```

Comment:
👍 thanks for the comments.

Reply:
NP, I thought especially that one was useful because of how flakey tests are reported